test/mindwendel/accounts_test.exs

```diff
@@ -79,5 +81,57 @@ defmodule Mindwendel.AccountsTest do
       Accounts.delete_inactive_users()
       assert Repo.exists?(from b in Brainstorming, where: b.id == ^old_brainstorming.id)
     end
+
+    test "does not delete the user is still attached to ideas", %{
+      old_user: old_user,
+      old_brainstorming: old_brainstorming
+    } do
+      Factory.insert!(:idea, brainstorming: old_brainstorming, user: old_user)
+
+      # we want to make sure that the database is not handling this with a foreign key restraint, but rather that it's handled in the app:
+      {_result, log} =
+        with_log(fn ->
+          Accounts.delete_inactive_users()
+        end)
+
+      assert Repo.exists?(from u in User, where: u.id == ^old_user.id)
+      assert log == ""
```

Comment:
I understand that you want to ensure the [`rescue` block](https://github.com/b310-digital/mindwendel/pull/192/files#diff-ce223e931ce628bfc48e506052c30fb93a1dfb2dd1764f774a6b46741567d0cbR138) is not called by checking for empty log entries.

It does not seem like the correct approach to me because log entires and application code can differ and are subject to change. I wonder if there is another approach that we can explore for ensuring the rescue block is not triggered.